app/views/groups/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="group-nav d-flex flex-row justify-content-between">
+ <%= link_to  groups_path, class: "link-add-cat ps-2" do %>
+    <i class="fa-sharp fa-solid fa-arrow-left top-link"></i>
+  <% end %>
+  <h3 class="nav-add-tras">TRANSACTIONS</h3>
+  <%= link_to destroy_user_session_path, data: {turbo_method: :delete}, class: "link-add-cat ps-2 top-link" do %>
+     <i class="fa-solid fa-arrow-right-from-bracket pe-2"></i>
+  <% end %>
+</div><br/>
+    <lu class="d-flex flex-column container-fluid">
+      <% @groups.each do |group| %>
+      <%= link_to group_path(group), class: "link-" do %>
+        <%= render "group", group: group%>
+        <% end %>
+      <% end %>
+    </lu>
+    <br/>
+<div class="container-fluid">
+  <div class="d-flex justify-content-center  btn-p">
+    <%= link_to "Add Item", new_group_path, class: "btn-prima" %>
+  </div>
+</div>
```

Comment:
- [ ] good job @george827, just a little fix andyou are good, according to the given design, the button to add item is at the bottom of the page, kindly make your page to look like the design given
your design :point_down: 
![Screenshot from 2023-04-28 11-45-44](https://user-images.githubusercontent.com/66097371/235130907-88ea6c8b-b7f5-4b54-a790-084459558764.png)

given design :point_down: 
![Screenshot from 2023-04-28 11-55-45](https://user-images.githubusercontent.com/66097371/235130758-be3ff113-024c-42b0-824e-989696f2a579.png)